### DISCUSSION TITLE
Peak2Gene links limited to one gene

### DISCUSSION THREAD

**Discussion by NoemieL** [GreenleafLab/ArchR#1470] [Questions / Documentation]
Hi,
I am interested in looking at peaks to a specific gene but in a big window. I adapted the upstream and downstream parameters in consequences. However I get the peak to gene links that are not related to the gene I mention in the geneSymbol parameter. How can I get rid of these links?

Thanks

**Comment by rcorces** [comment]
Sorry. I dont understand the question. I dont know what you want to do and I dont know what you mean by "the geneSymbol parameter". Please provide an example using the tutorial data if you would like help.

**Reply by NoemieL** [reply]
Sorry Ryan if I was not clear. I want the peak2Gene links that are specific to a gene. By increasing the window to get the peak2Gene that could be far from this gene, I also get the peak2Gene that are associated to other genes from this window. So I get a lot of unwanted peak2Gene links. How could I get rid of these unwanted links?

For exemple, I am interested by the gene MCL1, as you could see on the attached track plot that I obtained with the following command, there are peak2Gene links that are associated to ADAMTSL4 or ENSA. How could I remove these links from the track plot?
"'
p2g <- getPeak2GeneLinks(
  ArchRProj = ArchRProject,
  corCutOff = 0.45,
  resolution = 10000,
  returnLoops = TRUE
)

p <- plotBrowserTrack(
  ArchRProj = ArchRProject, 
  groupBy = "ID", 
  geneSymbol = "MCL1", 
  upstream = 50000,
  downstream = 50000,
  loops = getPeak2GeneLinks(ArchRProject),
  sizes = c(10, 1,2,2),
  pal=pal_ID
)
"'
![image](https://user-images.githubusercontent.com/26926109/173391759-4c73f2a9-d017-4b46-9c77-8c2257597215.png)

**Reply by rcorces** [reply]
just take the output of `getPeak2GeneLinks(ArchRProject)`, subset it by loops that overlap your promoter or region of interest, and pass that `plotBrowserTrack()`

**Reply by NoemieL** [reply]
Thanks a lot, I will try that!

**Comment by YesseniaCQ** [comment]
> Thanks a lot, I will try that!

Hi,did you kown how to achieve now?
